test/e2e/utils_test.go

```diff
@@ -1651,31 +1651,61 @@ func createNamespaceOnlyCRP(crpName string) {
 	createCRPWithApplyStrategy(crpName, nil, namespaceOnlySelector())
 }
 
-// ensureUpdateRunDeletion deletes the update run with the given name and checks all related approval requests are also deleted.
-func ensureUpdateRunDeletion(updateRunName string) {
+// ensureClusterStagedUpdateRunDeletion deletes the cluster staged update run with the given name and checks all related cluster approval requests are also deleted.
+func ensureClusterStagedUpdateRunDeletion(updateRunName string) {
 	updateRun := &placementv1beta1.ClusterStagedUpdateRun{
 		ObjectMeta: metav1.ObjectMeta{
 			Name: updateRunName,
 		},
 	}
 	Expect(client.IgnoreNotFound(hubClient.Delete(ctx, updateRun))).Should(Succeed(), "Failed to delete ClusterStagedUpdateRun %s", updateRunName)
 
-	removedActual := updateRunAndApprovalRequestsRemovedActual(updateRunName)
+	removedActual := clusterStagedUpdateRunAndClusterApprovalRequestsRemovedActual(updateRunName)
 	Eventually(removedActual, eventuallyDuration, eventuallyInterval).Should(Succeed(), "ClusterStagedUpdateRun or ClusterApprovalRequests still exists")
 }
 
-// ensureUpdateRunStrategyDeletion deletes the update run strategy with the given name.
-func ensureUpdateRunStrategyDeletion(strategyName string) {
+// ensureStagedUpdateRunDeletion deletes the staged update run with the given name and checks all related approval requests are also deleted.
+func ensureStagedUpdateRunDeletion(updateRunName, namespace string) {
+	updateRun := &placementv1beta1.StagedUpdateRun{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      updateRunName,
+			Namespace: namespace,
+		},
+	}
+	Expect(client.IgnoreNotFound(hubClient.Delete(ctx, updateRun))).Should(Succeed(), "Failed to delete StagedUpdateRun %s", updateRunName)
+
+	removedActual := stagedUpdateRunAndApprovalRequestsRemovedActual(updateRunName, namespace)
+	Eventually(removedActual, eventuallyDuration, eventuallyInterval).Should(Succeed(), "StagedUpdateRun or ApprovalRequests still exists")
+}
+
+// ensureClusterUpdateRunStrategyDeletion deletes the cluster update run strategy with the given name.
+func ensureClusterUpdateRunStrategyDeletion(strategyName string) {
 	strategy := &placementv1beta1.ClusterStagedUpdateStrategy{
 		ObjectMeta: metav1.ObjectMeta{
 			Name: strategyName,
 		},
 	}
 	Expect(client.IgnoreNotFound(hubClient.Delete(ctx, strategy))).Should(Succeed(), "Failed to delete ClusterStagedUpdateStrategy %s", strategyName)
-	removedActual := updateRunStrategyRemovedActual(strategyName)
+	removedActual := clusterUpdateRunStrategyRemovedActual(strategyName)
 	Eventually(removedActual, eventuallyDuration, eventuallyInterval).Should(Succeed(), "ClusterStagedUpdateStrategy still exists")
 }
 
+func ensureStagedUpdateRunStrategyDeletion(strategyName, namespace string) {
```

Comment:
Ditto, can we merge `ensureClusterUpdateRunStrategyDeletion` and `ensureStagedUpdateRunStrategyDeletion` into one and reuse the Actual function?